pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/util/UserUtil.java

```diff
@@ -298,6 +324,11 @@ public static List<String> getUserRolesPerRealm(@NonNull String query, @NonNull
         return roles;
     }
 
+    public static boolean checkGPUserAccessForPatch(String realm, String userId, String userIdRequest, Patch patch) {
+        return UserUtil.checkUserAccessForPatch(realm, userId, DBConstants.KIT_SHIPPING, userIdRequest, patch)
+                && DBConstants.DDP_KIT_ALIAS.equals(patch.getTableAlias());
+    }
```

Comment:
GP can only change kit values

Reply:
That's the role they already have, and one of the reasons i am making the change for this role is so they can all change the values if they have access

the only patch value for kit currently is the `collection_date`